harmony_py/config.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from dotenv import load_dotenv
+from typing import cast
+
+
+load_dotenv()
```

Comment:
If this module is imported, it'll try to find a .env file and load it, which could fail and throw an exception. I'd suggest moving this out of module scope and make it explicit behavior that happens as a side-effect of creating the Config, for example.